osquery/tables/system/ubuntu/deb_packages.cpp

```diff
@@ -179,7 +199,7 @@ QueryData genDebs(QueryContext &context) {
   dpkg_setup(&packages);
   for (int i = 0; i < packages.n_pkgs; i++) {
     struct pkginfo *pkg = packages.pkgs[i];
-    if (pkg->status == pkg->stat_notinstalled) {
+    if (pkg->status == PKG_STAT_NOTINSTALLED) {
```

Comment:
This is actually throwing a warning on older dpkg builds. Probably need to #if this as well.  Thoughts?